src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadId.cs

```diff
@@ -31,6 +35,29 @@ public WorkloadId(string id)
 
         public override string ToString() => _id;
 
+        public string ToSafeId() => ToSafeId(includeVisualStudioPrefix: false);
+
+        public string ToSafeId(bool includeVisualStudioPrefix)
```

Comment:
Is there a reason to have two separate overloads instead of just one with an optional parametr?

```suggestion
        public string ToSafeId(bool includeVisualStudioPrefix = false)
```

Reply:
No, I'll fix that.